### PR TITLE
A002210

### PR DESCRIPTION
**Pull request by s4mking**
Add 3 functions

### CODE REVIEW

**Comment by JulienPalard** [comment]
closes #20

**Comment by s4mking** [comment]
Change HEAD

File: oeis.py [hunk 1]
```diff
@@ -169,6 +151,27 @@ def phi(n):
     return [phi(x) for x in range(start, start + limit)]
 
 
+def A000040(start=0, end=999, plot=False):
```
Comment (JulienPalard) [inline oeis.py]: Missing: `@oeis` decorator to register it to the list of sequences.
Missing: A docstring indicating which sequence it is (see "oeis.py --list).

**Comment by JulienPalard** [comment]
Please also add some tests.

**Comment by s4mking** [comment]
Add documentation and testsing

File: oeis.py [hunk 2]
```diff
@@ -171,6 +153,29 @@ def phi(n):
...
+                if (val % n) == 0:
+                    break
+            else:
+                result.append(val)
+                resultIndex.append(i)
+                i = i + 1
+    if plot:
+        plt.plot(resultIndex, result)
+        plt.ylabel("some numbers")
+        plt.show()
```
Comment (JulienPalard) [inline oeis.py]: Functions does no longer accept `plot` since 27b2f2c.

**Pull request merged**